src/lib/environment.js

```diff
@@ -12,6 +11,7 @@ export const {
   WEBPACK_DEVTOOL,
 } = process.env
 
+export const ANALYZE_BUNDLE = process.env.ANALYZE_BUNDLE === "true"
```

Comment:
Why move out of the destructured process.env object just above?

Reply:
Just wanted to have it exported as a boolean. The switch statement in the index.js of the webpack config switch off of boolean true... A string wouldn't work.

Reply:
Oh that makes sense -- would need to be cased from process.env